data/include/actsvg/data/odd_pixel_endcap.hpp

```diff
@@ -19,288 +19,14 @@ namespace actsvg {
-    {-141.572, -100.789, -1524.4},
-    {-73.6854, -75.5362, -1523.2},
-    {-87.1839, -59.4492, -1523.2},
-    {-143.132, -98.5625, -1523.2},
-    {-121.92, -123.842, -1523.2},
-    {-59.4492, -87.1839, -1524.4},
-    {-75.5362, -73.6854, -1524.4},
-    {-123.842, -121.92, -1524.4},
-    {-98.5625, -143.132, -1524.4},
-    {-43.4067, -96.1827, -1523.2},
-    {-61.5933, -85.6827, -1523.2},
-    {-100.789, -141.572, -1523.2},
-    {-72.2106, -158.072, -1523.2},
-    {-26.0453, -102.259, -1524.4},
-    {-45.7789, -95.0765, -1524.4},
-    {-74.6744, -156.923, -1524.4},
-    {-43.6646, -168.21, -1524.4},
-    {-7.89258, -105.228, -1523.2},
-    {-28.5735, -101.582, -1523.2},
-    {-46.2905, -167.507, -1523.2},
-    {-13.7918, -173.237, -1523.2},
-    {10.5, -105, -1524.4},
-    {-10.5, -105, -1524.4},
-    {-16.5, -173, -1524.4},
-    {16.5, -173, -1524.4},
-    {28.5735, -101.582, -1523.2},
-    {7.89258, -105.228, -1523.2},
-    {13.7918, -173.237, -1523.2},
-    {46.2905, -167.507, -1523.2},
-    {45.7789, -95.0765, -1524.4},
-    {26.0453, -102.259, -1524.4},
-    {43.6646, -168.21, -1524.4},
-    {74.6744, -156.923, -1524.4},
-    {85.6827, 61.5933, -1524.4},
-    {96.1827, 43.4067, -1524.4},
-    {158.072, 72.2106, -1524.4},
-    {141.572, 100.789, -1524.4},
-    {61.5933, -85.6827, -1523.2},
-    {43.4067, -96.1827, -1523.2},
-    {72.2106, -158.072, -1523.2},
-    {100.789, -141.572, -1523.2},
-    {75.5362, -73.6854, -1524.4},
-    {59.4492, -87.1839, -1524.4},
-    {98.5625, -143.132, -1524.4},
-    {123.842, -121.92, -1524.4},
-    {87.1839, -59.4492, -1523.2},
-    {73.6854, -75.5362, -1523.2},
-    {121.92, -123.842, -1523.2},
-    {143.132, -98.5625, -1523.2},
-    {96.1827, -43.4067, -1524.4},
-    {85.6827, -61.5933, -1524.4},
-    {141.572, -100.789, -1524.4},
-    {158.072, -72.2106, -1524.4},
-    {102.259, -26.0453, -1523.2},
-    {95.0765, -45.7789, -1523.2},
-    {156.923, -74.6744, -1523.2},
-    {168.21, -43.6646, -1523.2},
-    {105.228, -7.89258, -1524.4},
-    {101.582, -28.5735, -1524.4},
-    {167.507, -46.2905, -1524.4},
-    {173.237, -13.7918, -1524.4},
-    {73.6854, 75.5362, -1523.2},
-    {87.1839, 59.4492, -1523.2},
-    {143.132, 98.5625, -1523.2},
-    {121.92, 123.842, -1523.2},
-    {59.4492, 87.1839, -1524.4},
-    {75.5362, 73.6854, -1524.4},
-    {123.842, 121.92, -1524.4},
-    {98.5625, 143.132, -1524.4},
-    {43.4067, 96.1827, -1523.2},
-    {61.5933, 85.6827, -1523.2},
-    {100.789, 141.572, -1523.2},
-    {72.2106, 158.072, -1523.2},
-    {26.0453, 102.259, -1524.4},
-    {45.7789, 95.0765, -1524.4},
-    {74.6744, 156.923, -1524.4},
-    {43.6646, 168.21, -1524.4},
-    {7.89258, 105.228, -1523.2},
-    {28.5735, 101.582, -1523.2},
-    {46.2905, 167.507, -1523.2},
-    {13.7918, 173.237, -1523.2},
-    {-10.5, 105, -1524.4},
-    {10.5, 105, -1524.4},
-    {16.5, 173, -1524.4},
-    {-16.5, 173, -1524.4}};
+extern std::vector<std::array<scalar, 3u>> odd_pixel_endcap;
```

Comment:
Also the same here. Does this need to be a public variable?

Reply:
That's for testing only, as there's also an input layer now, we can probably remove the data all along.

Reply:
I'll leave this as-is if it can be removed soon.